paasta_tools/kubernetes_tools.py

```diff
@@ -486,12 +495,40 @@ def get_autoscaling_params(self) -> AutoscalingParamsDict:
             defaults=default_params,
         )
 
+    # TODO: move the default scaling policy to system paasta configs
+    # TODO: allow per-instance overrides in soa-configs, and merge this function
+    #   with `get_autoscaling_params`
+    def get_autoscaling_scaling_policy(self, max_replicas: int) -> Dict:
+        """Returns the k8s HPA scaling policy in raw JSON. Requires k8s v1.18
+        to work.
+        """
+        # policy in human terms: scale down at most every 60 seconds by 30% of
+        # current replicas or 5% of max replicas, whichever is higher.
```

Comment:
Just out of curiosity, how did we come to this?

Reply:
The 30% I picked from the PaaSTA service autoscaler, which I believe had a scaling limit of 30% of current replicas. The 5% is an arbitrary low end I picked. If we wanted to simplify, we could just remove the "Pods" policy and just keep the "Percent" policy.

Reply:
I would be in favor of simplifying it. In the project meeting this week, we were talking about how HPA doesn't make it easy to understand the decisions that it makes, through its logging, so we'd rather have as many of the decisions visible through our metrics instead.

Reply:
Done.